readme/Configuration.scalatex

```diff
@@ -381,6 +381,26 @@
       implicit def validatedInstances[E](implicit E: Semigroup[E]): Traverse[
           Validated[E, ?]] with ApplicativeError[Validated[E, ?], E] = 2
 
+  @sect{newlines.beforeCaseClassParameterDefn}
+    Together with @b{verticalMultilineAtDefinitionSite}, this will
+    align parameter definitions vertically, with a line for each, and
+    not attempt to fit the case class definition on a single line.
+
+    Default @b(default.newlines.beforeCaseClassParameterDefn)
+
+    Requires @b{verticalMultilineAtDefinitionSite} set to @b(true).
+
+    @hl.scala
+      // verticalMultilineAtDefinitionSite = true
```

Comment:
Why is this required? Seems like this option should be usable without  verticalMultilineAtDefinitionSite=true.

Reply:
Actually, I think this should be the default behavior of **verticalMultilineAtDefinitionSite** :-)

It seems inconsistent to me, that **verticalMultilineAtDefinitionSite** does not do vertical alignment (as specifically asked), if the line is short enough. This is the default behavior now, and I don't expect to get that changed, which is why I added a new setting.

If I make my new setting distinct from  **verticalMultilineAtDefinitionSite**, wouldn't we just have to settings, basicly doing the same, only with different behaviors in the single-line case?